### DISCUSSION TITLE
Alle Verkehrsunternehmen bei Statistik

### DISCUSSION THREAD

**Discussion by Lucasffca** [Traewelling/traewelling#2324] [Ideas / Feature Requests]
DE: 
Moinsen liebe Traeweller, 
ich würde mir wünschen in Zukunft in der Statistik alle Verkehrsunternehmen aufgelistet zu bekommen, anstatt dass die am wenigsten benutzten unter Sonstige aufgeführt werden.

Wenn dies schlecht umsetzbar ist, wäre es zumindest Toll, wenn man eine Extra Auflistung für alle bisher genutzten Eisenbahnverkehrunternehmen hätte, da ich gerne mal sehen möchte welche ich schon mal gefahren bin oder eben nicht.

Liebe Grüße 
Lucas aus dem schönen Hessen

EN:
Hello dear Traewellers,
I would like to see all transport companies listed in the statistics in the future, instead of the least used ones being listed under Other.

If this is difficult to implement, it would at least be great if there was a separate list for all the railway companies that I have used so far, as I would like to see which ones I have used or not.

Best regards
Lucas from beautiful Hesse

**Comment by MrKrisKrisu** [comment]
> in der Statistik alle Verkehrsunternehmen aufgelistet zu bekommen, anstatt dass die am wenigsten benutzten unter Sonstige aufgeführt werden.

Tatsächlich fassen wir da nichts zusammen. Unter "Sonstige" fallen alle Verkehrsunternehmen, die uns nicht bekannt sind. Die Fahrplandaten die wir nutzen (HAFAS der DB) hat nicht zu jeder Fahrt einen entsprechenden Operator.
